File: common/src/main/java/generations/gg/generations/core/generationscore/common/world/level/block/IPortalBlock.java
package generations.gg.generations.core.generationscore.common.world.level.block;

import net.minecraft.core.BlockPos;
import net.minecraft.world.level.BlockGetter;
import net.minecraft.world.level.block.state.BlockState;

public interface IPortalBlock {
    boolean isPortalFrame(BlockState state, BlockGetter level, BlockPos pos);
}

File: src/common/main/kotlin/generations/gg/generations/core/generationscore/common/world/level/block/IPortalBlock.kt
package generations.gg.generations.core.generationscore.common.world.level.block

import net.minecraft.core.BlockPos
import net.minecraft.world.level.BlockGetter
import net.minecraft.world.level.block.state.BlockState

interface IPortalBlock {
    fun isPortalFrame(state: BlockState, level: BlockGetter, pos: BlockPos): Boolean
}
